### ISSUE TITLE
No longer functioning on my chrome browser

### ISSUE THREAD

**Issue by aha2015** [Mattie432/YouTweak#16]
For some reason the extension no longer works on my browser. I can still go in and update preferences settings, but my subscriptions page looks the same as always. It has been working for months and I quite love it, so this is making me very sad. :-( Any help would be greatly appreciated. I've already tried restarting, btw :-)

**Comment by Mattie432** [comment]
The branch 2015-10-16_YouTubeUpdate is addressing a similar issue ( #15 ). Could you download and test [this crx](http://www.mediafire.com/download/evgv49mz8yufmyu/chrome.crx) using [this tutorial](https://github.com/Mattie432/YouTweak/wiki/installing-a-developer-build). Should be able to remove videos again if it is the same issue. Thanks.

**Comment by DemonAMVs** [comment]
Tried to get the CRX working, enabled developer mode, installed extension.  In options I enabled both redirections, as well as add the remove button.  Hit save and nothing appears on the youtube subscription page, does not redirect.  If I enable removed watched videos, that will appear but will not do anything when clicked.

Second time I enabled everything, same result except clear all videos, and load all videos did appear, they are not functional, load all videos did bring up the dialogue.

**Comment by aha2015** [comment]
I used the tutorial and when I went to drop to install nothing happened. If it's useful to know, when downloading the extension a pop up on chrome came that said that chrome won't allow extensions from that site. I had already enabled developer mode.

**Comment by aha2015** [comment]
Nevermind, I figured out how to work the extension. The remove button is back. Thank you!

**Comment by Mattie432** [comment]
Okay, good to know. Please not this is just a developer build to check that the fix's work. I am trying to push out the official release asap.

**Comment by Mattie432** [comment]
Could you please test the pre-release build [here](https://github.com/Mattie432/YouTweak/releases/tag/3.234). Especially the "version selection" option, which should work for you when selecting 'latest' and not work when selecting 'previous'.

Thanks.

**Comment by DemonAMVs** [comment]
Testing now, turning on all options with latest build options, similar to previous version.
On subscriptions page
Remove Video buttons: Not appearing
Remove all watched button: did not appear.
Load all videos to sub page: Functions as intended
Clear all videos: Not willing to test, this is a feature I do not intend to use.
Redirection: Functioning, some of the time
Remove recommended channels: I saw none so I presume it works.


On Home page.
Remove Video buttons: appearing, and functioning.
Remove all watched button: did not appear.
Load all videos to sub page: NA.
Clear all videos: NA
Redirection: Functioning, some of the time
Remove recommended channels: I saw none so I presume it works.

Summary: The main feature I use which is the remove video button functions which works as intended on the home page, but not the subscription page.

**Comment by DemonAMVs** [comment]
With Previous option enabled.
Autolike: Not functioning


On subscriptions page
Remove Video buttons: Do not appear
Remove all watched button: Did not appear.
Load all videos to sub page: Functions as intended
Clear all videos: Not willing to test, this is a feature I do not intend to use.
Redirection: Working as intended
Remove recommended channels: I saw none so I presume it works.

On Home page.
Remove Video buttons: appearing, and functioning.
Remove all watched button: did not appear.
Load all videos to sub page: NA.
Clear all videos: NA
Redirection: Working as intended
Remove recommended channels: I saw none so I presume it works.
